samples/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM alpine:3.7
+RUN apk add --no-cache git git-lfs
+
+VOLUME [ "/data" ]
```

Comment:
I don't see the .gitignore checked in for `/data`.

Reply:
good one! fixt!